src/rpc/net.cpp

```diff
@@ -491,6 +492,36 @@ static UniValue getnetworkinfo(const JSONRPCRequest& request)
     return obj;
 }
 
+static UniValue getnetworkrpcinfo(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw std::runtime_error(
+            "getnetworkrpcinfo\n"
+            "Returns an object containing various state info regarding RPC service.\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"total_requests\": xxxxx,            (numeric) Lifetime total number of RPC requests processed.\n"
+            "  \"uptime_secs\": xxxxx,               (numeric) Uptime of RPC server in seconds.\n"
+            "  \"total_request_time_micros\": xxxxx  (numeric) Total time spent processing RPC requests in microseconds.\n"
```

Comment:
Is this relevant?

Reply:
If the uptime is recorded in micros, then I think you are right that this becomes redundant.